src/test/java/com/okta/tools/aws/settings/CredentialsTest.java

```diff
@@ -35,19 +34,10 @@
     private String accessKey = "accesskey";
     private String secretKey = "secretkey";
     private String sessionToken = "sessiontoken";
-    private String manualRole = "[" + roleName + "_source]\n"
```

Comment:
:fire: the "_source" suffix was a workaround for improperly writing ~/.aws/config so it is no longer the default.